oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -393,6 +393,7 @@ receiver-zipkin:
     jettyAcceptorPriorityDelta: ${SW_RECEIVER_ZIPKIN_JETTY_DELTA:0}
     jettyAcceptQueueSize: ${SW_RECEIVER_ZIPKIN_QUEUE_SIZE:0}
     instanceNameRule: ${SW_RECEIVER_ZIPKIN_INSTANCE_NAME_RULE:[spring.instance_id,node_id]}
+    searchableTracesTags: ${SW_ZIPKIN_SEARCHABLE_TAG_KEYS:http.method}
```

Comment:
Add this to the doc please

Reply:
I have added this and some missing items to the doc.  I found the env. variable still name `JETTY_xx`, should we keep these name styles for compatibility?

Reply:
See https://github.com/apache/skywalking/pull/8971#discussion_r861007722